### DISCUSSION TITLE
`left-focused-column`, keep any focused window to the left

### DISCUSSION THREAD

**Discussion by phanirithvij** [YaLTeR/niri#694] [Ideas]
I wish to have the focused window to shift left to the screen always, whilemoving left/right.

Currently when I do mod+left, focus is behaving like I want to, stays on left, and mod+right has the same behavior (window stays to the right edge), I want the moving windows to stick to the left edge.

I've tried as per the suggestion in [matrix](https://matrix.to/#/!KIjqiaZyJFkPXxMmGQ:gnome.org/$QwYyXZ13SK-syJqW5dFZ3Zk2HOCq_rmWYnHVom3CBmY?via=gnome.org&via=matrix.org&via=matrix.opulus.space)

binding to Mod+Left with `spawn bash focus-left.sh`
```bash
niri msg action focus-column-left # -right for Mod+Right
niri msg action maximize-column
niri msg action maximize-column
```

But it is pretty jarring and bouncy.

**Comment by YaLTeR** [comment]
I guess it would make sense to at least add actions analogous to `center-column` to pad a column left or right. Not sure about an always-left setting yet, that requires an annoying amount of changes.
